king_phisher/client/plugins.py

```diff
@@ -543,3 +552,208 @@ class ClientPluginManager(plugins.PluginManagerBase):
 	_plugin_klass = ClientPlugin
 	def __init__(self, path, application):
 		super(ClientPluginManager, self).__init__(path, (application,))
+
+class RepoCache(object):
```

Comment:
`RepositoryCache` so it's a consistent mapping to `catalog.Repository` like `CatalogCache` is to `catalog.Catalog`.